doc/updating-sdk.md

```diff
@@ -50,4 +50,5 @@ SDK:
 
 ```bash
 vendor/bin/jane-openapi generate --config-file=.jane-openapi.php
+vendor/bin/php-cs-fixer fix
```

Comment:
Not needed if you do cs-fix during Jane generation (look at https://github.com/jolicode/slack-php-api/pull/73/files#r382296204)